playbooks/installation/installation.adoc

```diff
@@ -50,28 +50,32 @@ The Smart Start architecture requires the following Servers(VMs) be provisioned:
 
 * 3 _Masters_
   ** RHEL 7.2 minimal installation
-  ** 8 GB Memory
-  ** 2 Cores
-  ** 30 GB Root Drive
-  ** An additional 5 GB disk or partition mounted at /var/lib/etcd
-* 2 _Infrastructure Nodes_
-  ** RHEL 7.2 minimal installation
-  ** 8 GB Memory
+  ** 20 GB Memory
   ** 4 Cores
-  ** 30 GB Root Drive
-  ** An additional 20 GB disk or partition mounted at /var/lib/origin
-  ** An additional 50 GB disk, unformatted and unmounted
+  ** 60 GB for the root (`/`) partition or logical volume containing `/var`
+  ** An additional 10 GB disk or logical volume mounted at `/var/lib/etcd`
+  ** An additional 50 GB disk or logical volume, unformatted and unmounted (for local Docker)
+* 3 _Infrastructure Nodes_
+  ** RHEL 7.2 minimal installation
+  ** 24 GB Memory
+  ** 6 Cores
+  ** 40 GB for the root (`/`) partition or logical volume containing `/var`
+  ** An additional 20 GB disk or logical volume mounted at `/var/lib/origin`
+  ** An additional 100 GB allocated to a volume group, unformatted and unmounted (for local Docker)
 * 3 _Application Nodes_
-  ** 16 GB Memory
+  ** 48 GB Memory
   ** 4 Cores
-  ** 30 GB Root Drive
-  ** An additional 20 GB disk or partition mounted at /var/lib/origin
-  ** An additional 50 GB disk, unformatted and unmounted
+  ** 30 GB for the partition or logical volume containing `/var`
+  ** An additional 20 GB logical volume mounted at `/var/lib/origin`
+  ** An additional 100 GB disk, unformatted and unmounted
 * (Optional) A Load Balancer host, if you plan to use Option 2 for Load Balancing, per the above section
   ** 2 cores
   ** 4 GB Memory
   ** 10 GB Root Drive
 
+TODO: Need something about persistent storage
+- push for iSCSI volumes for registry/metrics/logging
+
 NOTE: We chose the resource sizing above based on some reasonable starting sizes for hosts based on our experiences in the field. Before building further environments, it is important to go through a real production sizing exercise. For more information on how we recommend going about this, see our link:/playbooks/operationalizing/environment_sizing{outfilesuffix}[Environment Sizing Guide].
 
 NOTE: We recommend segregating local disks this way in order to provide more stability to the separate components that will write to these locations. For more information about why we configure local storage this way, read link:/playbooks/operationalizing/local_storage{outfilesuffix}[Considerations for Configuring Local Storage on OpenShift Hosts]
```

Comment:
line 100  below - replace 'two' with 'three' in the 4th bullet point for the infrastructure nodes